src/components/MovieCard/MovieCard.tsx

```diff
@@ -1,7 +1,41 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  film: Movie;
+};
```

Comment:
You can use you Movie type as a Props type
```suggestion
type Props =  Movie;
```

Reply:
this one doesn't work